CHANGES.md

```diff
@@ -9,6 +16,36 @@
   runtime overhead, but may be disabled using the `--no-call-stacks`
   command-line option.
 
+* The `:exhaust` command now works for floating-point types.
+
+* The `:check` command now uses more representative sampling of random
+  input values to test.
```

Comment:
This is also WRT floating-point values, I think?

Reply:
Ah, good. Thanks for clarifying that. I'll update the text.